egs2/librimix/sot_asr1/local/metadta_Libri2Mix_offset/libri2mix_dev-clean.csv

```diff
@@ -0,0 +1,3001 @@
+mixture_ID,source_1_path,source_1_gain,source_2_path,source_2_gain,noise_path,noise_gain,offset
```

Comment:
can you explain why this is needed?
to avoid the full overlap in the original librimix?

Reply:
Yes, the updated csv file includes an additional column of "offset", which is used as the time delay when simulating overlapped data. Here we randomly generate offsets from 1s-1.5s.
The motivation for fixing the offset is to allow people to do a fair comparison on the same simulated data among different models.